server/routes/termdb.getdescrstats.ts

```diff
@@ -84,7 +84,7 @@ async function trigger_getdescrstats(q: any, res: any, ds: any, genome: any) {
 	for (const key in data.samples) {
 		const sample = data.samples[key]
 		const value = sample[q.tw.$id].value
-		if (q.tw.values?.[value]?.uncomputable) {
+		if (q.tw.q.hiddenValues?.[value]) {
```

Comment:
@gavrielm can u doublecheck this? hiddenValues is not part of standard tw.q{}

Reply:
Looks to be correct. `hiddenValues` is defined in `BaseQ` in `client/shared/types/terms/term.ts` and is defined in `tw.q` when running both barchart and violin plot.